javascript-objects/main.js

```diff
@@ -0,0 +1,30 @@
+const student = {
+  firstName: 'Kitlong',
+  lastName: 'Lai',
+  age: 26
+};
+const fullName = student.firstName + student.lastName;
```

Comment:
This will evaluate to `KitlongLai` - we should add a `space` character in this string concatenation.